src/lsst/cmservice/web_app/app.py

```diff
@@ -1,31 +1,29 @@
-from pathlib import Path
```

Comment:
Oops -- looks like isort (ruff-format) ran when I edited this file.  I will see about backing out those changes -- they are probably best done wholesale in `web_app/` on a separate commit to avoid merge conflicts.